docs/verifiable_credentials_examples/talent_protocol_passports/panagiotis_matsinopoulos.json

```diff
@@ -0,0 +1,63 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "https://passport.talentprotocol.com/passports/0x324e9E13dd19528D0F390201923d17c4B7E94462",
+  "type": [
+    "VerifiableCredential",
+    "TalentProtocolPassportCredential"
+  ],
+  "name": "Talent Protocol Credentials",
+  "description": "A verification of Talent Protocol Credentials issued by Talent Protocol to Builders",
+  "issuer": {
+    "id": "https://passport.talentprotocol.com/credentials/issuers/talentprotocol.eth",
+    "name": "Talent Protocol"
+  },
+  "credentialSubject": {
+    "id": "https://etherscan.io/address/0x324e9e13dd19528d0f390201923d17c4b7e94462",
+    "name": "Panagiotis Matsinopoulos", // custom
+    "ens": {
+      "id": "https://passport.talentprotocol.com/credentials/02f9d83f-32b9-4782-83f1-bed7f863c5fc", // passpord_credentials#uuid
+      "name": "PassportCredentials::Ens", // custom
+      "category": "Identity", // custom
+      "validFrom": "2024-04-14T07:12:35Z", // custom taken from onchain_at ? or other datetime
+      "validUntil": null, // custom
+      "score": 5, // custom
+      "data": "{\"primary_domain\":\"panagiotismatsinopoulos.eth\",\"domains\":[\"panagiotismatsinopoulos.eth\"]}", // custom
+    },
+    "takeOff": {
+      "id": "https://passport.talentprotocol.com/credentials/5adf0e4a-f10f-4168-ba75-cfce78cb9967", // passpord_credentials#uuid
+      "name": "PassportCredentials::TakeOff", // custom
+      "category": "Skills", // custom
+      "validFrom": "2024-08-07T12:34:36Z", // custom taken from last calculated at or other date time
+      "validUntil": null, // custom
+      "score": 9, // custom
+      "data": "{\"edition\":\"#12 Brussels Edition\"}", // custom
+    }
+  },
+  "validFrom": "2024-08-01", // this should be the earliest +datetime+ from the credentialSubject#validFrom
+  "validUntil": null, // this should be the latest +datetime+ from the credentialSubject#validFrom
+  "status": [{
+    "type": "TalentProtocolCredentialStatus",
+    "credentialSubject": { // custom
+      "id": "https://passport.talentprotocol.com/credentials/02f9d83f-32b9-4782-83f1-bed7f863c5fc"
+    },
+    "revoked": false // custom
+  }, {
+    "type": "TalentProtocolCredentialStatus",
+    "credentialSubject": { // custom
+      "id": "https://passport.talentprotocol.com/credentials/5adf0e4a-f10f-4168-ba75-cfce78cb9967"
+    },
+    "revoked": false // custom
+  }],
+  "credentialSchema": {
+    "id": "https://passport.talentprotocol.com/schemas/TalentProtocolCredential.json",
+    "type": "JsonSchema"
+  },
+  "termsOfUse": {
+    "id": "https://passport.talentprotocol.com/credentials/terms_of_use",
+    "type": "IssuerPolicy"
+  },
+  "evidence": [{...}, {...}] // do we want to provide objects of evidence? https://www.w3.org/TR/vc-data-model-2.0/#evidence
```

Comment:
i think we can have a list of transactions here that were checked to create the credentials, but not sure if it’s worth the extra work? what do you think?

Reply:
> i think we can have a list of transactions here that were checked to create the credentials, but not sure if it’s worth the extra work? what do you think?

Do we have this piece of information for each credential type? And can we present them in a unified way? Maybe we can skip for the Hackathon.